core/src/credential.rs

```diff
@@ -38,41 +124,36 @@ impl CredentialType {
             "BankAccount" => Some(CredentialType::BankAccount),
             "PermanentResidentCard" => Some(CredentialType::PermanentResidentCard),
             "VerifiableCredential" => Some(CredentialType::Common),
+            "UniversityDegreeCredential" => Some(CredentialType::UniversityDegreeCredential),
+            "AlumniCredential" => Some(CredentialType::AlumniCredential),
             _ => None,
```

Comment:
I like to write this sort of pattern as:
```
Some(match cred_type => {
   "BankCard => CredentialType::BankCard
   ...
   _ => return None
   })
```

since it reduces the number of times you have to write out `Some`

Reply:
@gshuflin That makes sense, also do you konw if there's a way to guarantee that there's no members of `CredentialType` left unmatched?

Reply:
My one issue with this approach is that it makes the return model more complicated/unclear

Reply:
Unrelated to this PR but we need to refactor this to avoid needing to hardcode the credentialTypes.  We need to be able to support various new cred types- ex DriversLicense, ProofOfEmployment, etc without needing to issue new SSI SDKs and clients needing to do new builds... Note Registry Mgmt will have all VC schema types so we can refer to that as well if required for schema definitions.